hirte.spec.in

```diff
@@ -1,7 +1,7 @@
 Name:		hirte
 Version:	@VERSION@
 Release:	@RELEASE@%{?dist}
-Summary:	Hirte is a service orchestrator tool intended for multi-node devices (e.g.: edge devices) clusters
+Summary:	Service orchestrator tool for multi-node devices (e.g.: edge devices)
```

Comment:
I would have started with `A`:
```suggestion
Summary:	A service orchestrator tool for multi-node devices (e.g.: edge devices)
```

Reply:
Inspired in systemd, which has:

```
Summary:        System and Service Manager
```

But of course I can change if you think that with 'A' it's better

Reply:
OK, 'A' added